src/_playground/Routes.js

```diff
@@ -270,59 +271,99 @@ const routes = [
     }
 ];
 
-export const Routes = () => {
-    let sectionRoutes;
-    const groupedRoutes = groupArray(routes, 'section');
+export class Routes extends Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            query: '',
+            filteredItems: routes
+        };
+    }
 
-    const navItems = sections.sort(sortBy('sortOrder', 'name')).map(section => {
-        if (!groupedRoutes[section.name]) {
-            return;
-        }
+    onChangeHandler = (event) => {
+        let searchResults = routes.filter((navItem) => {
+            return navItem.sortOrder === 1 ? navItem : navItem.name.toLowerCase().includes(event.target.value.toLowerCase());
+        });
+        this.setState({
+            query: event.target.value,
+            filteredItems: [...searchResults]
+        });
+    };
 
-        sectionRoutes = groupedRoutes[section.name].sort(sortBy('sortOrder', 'name'));
-        return (
-            <ul className='frDocs-Nav__list' key={section.name}>
-                <li className='frDocs-Nav__headers'>{section.name}</li>
-                {sectionRoutes.map(route => (
-                    <li key={route.name}>
-                        <NavLink
-                            activeClassName='frDocs-Nav__item--active'
-                            className='frDocs-Nav__item'
-                            key={route.url}
-                            to={{ pathname: route.url }}>
-                            {route.name}
-                        </NavLink>
-                    </li>
-                ))}
+    render() {
+        let sectionRoutes;
+        let routedItems;
+        this.state.query === '' ? routedItems = routes : routedItems = this.state.filteredItems;
```

Comment:
I don't _think_ you need to do this.  You are storing the routes in the `filteredItems` in state so you should just always be able to use that.

Reply:
You're right. This is an artifact from a different implementation, so I've removed it now. It should now be fixed in latest push.